### ISSUE TITLE
SchemaSpy: Add navigable Schema diagrams at each release 

### ISSUE THREAD

**Issue by christinerogers** [aces/Loris#5391]
Based on example presented from CNBP by @dyt811
https://cnbp.github.io/LORIS_MySQLStructure/

Should be set up as an automatic process to execute on the schema and be committed as release is issued. 
Web front-end accessibility of this documentation should be tested as part of the release process.

**Comment by johnsaigle** [comment]
Related #5880

**Comment by christinerogers** [comment]
@Haowei -- I'm repurposing this Issue for you as a starting point for the Database Schema Diagram (ERD): 
This is a good next step in your search for software that will automatically generate a diagram based on our schema, which should trigger ideally through Github at each release. 
We have a colleague on the CBNP project who had success with using SchemaSpy to automatically generate a summary (linked above) and a diagram of database .  
Take a little time to investigate and let me know what you think -- 
There are other software tools out there for this kind of task, as well, but check out SchemaSpy first.

**Comment by haoweiqiu** [comment]
Hi @christinerogers, sounds great! I'll look into this. Thank you!
